### DISCUSSION TITLE
1

### DISCUSSION THREAD

**Discussion by ghost** [WeNeedHome/SummaryOfLoanSuspension#577] [Ideas]

**Comment by zxcs001** [comment]
对的，我们就是乌合之众啊。

我们就是躺平的垃圾，还钱？一分钱没有。

你知道什么叫做无敌的人么？无所顾忌，烂命一条，你最好祈祷这些人聚集起来的时候，到时候把你的狗头给你砸烂

**Reply by dongyc519** [reply]
道理没错，走投无路最可怕。
